create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/app.py

```diff
@@ -33,19 +33,22 @@ def init_jinja2(app: web.Application) -> None:
 {%- if cookiecutter.use_postgres == 'y' %}
 
 
-async def init_database(app: web.Application) -> None:
+async def database(app: web.Application) -> None:
     '''
     This is signal for success creating connection with database
```

Comment:
U need update docstring. Now it's not only `startup` function 😀

Reply:
@Arfey done